### ISSUE TITLE
Better document the mount behavior in appsody

### ISSUE THREAD

**Issue by kylegc** [appsody/website#233]
Follow-up to https://github.com/appsody/stacks/issues/280
We should do a better job of documenting the mount points, specifically that the host path is relative to the appsody project dir.

**Comment by skoh7645** [comment]
Do we have an idea as to where we would add this?

**Comment by kylegc** [comment]
@appsody/website please see Sandy's question above.

**Comment by Kamran64** [comment]
We could document this in the [stack structure](https://appsody.dev/docs/stacks/stack-structure) page under or near **Mounting the correct directories into the image**. My only worry is that this page is already very text heavy as it is...

**Comment by tomleah** [comment]
> My only worry is that this page is already very text heavy as it is...

Agreed, I couldn't even find the heading you were talking about @Kamran64, its all 1 block of text. We should at improving the doc pages design.

We might need some more explanation in [Appsody Environment Variables](https://appsody.dev/docs/stacks/environment-variables) also? Not as much how the mounts work, but just verify how mounts must be specified. I see some conversation about `.` vs `/` in https://github.com/appsody/stacks/issues/280

Another idea, maybe we start on getting some architecture diagrams in a high-level architecture  doc. Then we can graphically show where directories are and what happens to them on `appsody init`, `appsody extract` etc. We can then show the relation between host and container directories with the ENV variables/mounts

**Comment by helenmasters** [comment]
@kylegc the env variables documentation has had an overhaul recently which addresses how the mounts must be specified. What else would you like to see documented given the docs we have now?

**Comment by kylegc** [comment]
I changed the first comment into a checklist and reviewed the documentation. From what I see, only one of the three items is covered by documentation. We still need the following:

**1) Mount host path is relative to the appsody project dir**
Update APPSODY_MOUNTS in [Appsody Environment Variables](https://appsody.dev/docs/stacks/environment-variables) to make this clear. It should explain what a `.`, `/`, and `~` means in the path.

**2) What will be extracted**
This should probably documented/improved in a few places:
- The CLI Reference/help text for extract
- The [Stacks Overview] (https://appsody.dev/docs/stacks/stacks-overview) page
- The APPSODY_PROJECT_DIR in [Appsody Environment Variables](https://appsody.dev/docs/stacks/environment-variables) needs improvement. Both the description and the example are misleading. This value is not a local path, it is a path within the container.
